tobac/segmentation.py

```diff
@@ -1079,6 +1091,8 @@ def segmentation(
     PBC_flag="none",
     seed_3D_flag="column",
     seed_3D_size=5,
+    below_threshold_num=0,
```

Comment:
you forgot to change the names here as well :)

Reply:
Good catches!